crane-cloud/frontend
# handle clearing of transactions

# Description

Project transactions that were fetched before were appearing in different projects. they needed clearing.

## Type of change



## Trello Ticket ID

https://trello.com/c/FJFQ0Ydk

## How Can This Been Tested?

pull the  branch and check out transactions of each project, they should be unique to a project


## Checklist:


# Screenshots
